tests/roots/test-latex-table/expects/complex_spanning_cell.tex

```diff
@@ -16,39 +19,51 @@
 \hline
 \sphinxmultirow{3}{1}{%
 \begin{varwidth}[t]{\sphinxcolwidth{1}{5}}
+
+\sphinxAtStartPar
```

Comment:
I wonder why my patch to ``visit_paragraph`` resulted in this context in extra blank line (we can see many instance in this file). Perhaps has to do with the ``needs_linetrimming`` conditional and the injection of ``\n`` at end of ``varwidth``. This is at any rate not a worry as it changes nothing to PDF output. I will make double-check nevertheless.